events.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+var cardId;
+var userInput = document.getElementById("userInput");
+
+var CarLot = (function(originalCarLot) {
+
+  originalCarLot.activateEvents = function() {
+
+    let cardListener = document.getElementsByClassName("carCard");
+
+    for (let i = 0; i < cardListener.length; i++) {
+       let cardThing = cardListener[i];
+
+       cardThing.addEventListener("click", CarLot.youClickedMe);
+    };
+  };
+
+  originalCarLot.youClickedMe = function() {
+
+    CarLot.changeBorder();
+    userInput.focus();
+
```

Comment:
Remember, per the requirements you also need to clear the text box when a card is clicked